examples/text-classification/run_xnli.py

```diff
@@ -36,10 +37,12 @@
     AutoConfig,
     AutoModelForSequenceClassification,
     AutoTokenizer,
+    HfArgumentParser,
+    TrainingArguments,
     get_linear_schedule_with_warmup,
 )
 from transformers import glue_convert_examples_to_features as convert_examples_to_features
-from transformers import xnli_compute_metrics as compute_metrics
+from transformers import set_seed
 from transformers import xnli_output_modes as output_modes
 from transformers import xnli_processors as processors
```

Comment:
Those shouldn't be imported and all preprocessing should be done in the example script, like for the new `run_glue`.

Reply:
ah i see; let me take another crack at it. If it proves to be too advanced for me, I'll just close the PR.

Reply:
Sure! And happy to help if you have some issues.